internal/metrics/metrics.go

```diff
@@ -362,14 +361,20 @@ func RegisterClientMetrics(r prometheus.Registerer) error {
 // Handler abstracts a helper for relaying http requests to a group peer
 type Handler func(ctx context.Context, addr string) (http.Handler, error)
 
-// Start starts a prometheus metrics server with debug endpoints.
-func Start(logger log.Logger, metricsBind string, pprof http.Handler, groupHandlers []Handler) net.Listener {
```

Comment:
is there a reason we don't take a handler for each beaconID now?

Reply:
Because we never had per-beaconID group metrics it seems: our group metrics seem to always have been global and encompass all beaconIDs at once, IIUC the code... 

But I might be wrong...

Reply:
on the dashboard I see some group size/threshold per beaconID, though all at once (regardless of the dropdown) - perhaps they're just all sent at once